openedx/features/student_account/forms.py

```diff
@@ -76,6 +84,21 @@ def clean_organization_name(self):
                 return org_name
         return org_name
 
+    def clean_g_recaptcha(self):
+        g_recaptcha = self.cleaned_data.get('recaptcha')
```

Comment:
Let's please choose the same naming convention across the functionality either `g_recaptcha` or `recaptcha`